### PR TITLE
feat: create endpoints for directory member details

### PR DESCRIPTION
**Pull request by AsiakN**

### CODE REVIEW

File: functions/package.json
Comment (jaedag) [inline functions/package.json:1]: Why editing the configs?

File: firebase.json
Comment (jaedag) [inline firebase.json:1]: What's the purpose of these edits?

**Review by jaedag — DISMISSED** [review]
Looks good. Wiill test it out later today

File: firebase.json
Comment (AsiakN) [reply firebase.json:1]: I want to run the emulators from another shell. So I have to set the ports for them.  The emulators don't start when you run them fwith npm run dev

**Comment by jaedag** [comment]
@AsiakN Still no looms to help me review?

**Pull request merged**